spec/tests/angularComponent/ngcOmniboxControllerSpec.js

```diff
@@ -414,6 +416,93 @@ describe('ngcOmnibox.angularComponent.ngcOmniboxController', () => {
     });
   });
 
+  describe('choosing and unchoosing', () => {
+    beforeEach(() => {
+      omniboxController.ngModel = ['one', 'two'];
+    });
+
+    it('should set the ngModel to the choice when multiple is off', () => {
+      omniboxController.choose('three');
+
+      expect(omniboxController.ngModel).toEqual('three');
+    });
+
+    it('should push the choice to an array when multiple is on', () => {
+      omniboxController.multiple = true;
```

Comment:
@onlywei or I get even more strict and say I'll only treat it as an array if `Array.isArray` _and_ `multiple` are `true`.